spring-cloud-gateway-core/src/main/java/org/springframework/cloud/gateway/config/HttpClientProperties.java

```diff
@@ -293,6 +293,36 @@ public String toString() {
 		/** The default ssl configuration type. Defaults to TCP. */
 		private SslProvider.DefaultConfigurationType defaultConfigurationType = SslProvider.DefaultConfigurationType.TCP;
 
+		private String keyCertChainPath;
+
+		private String keyFilePath;
+
+		private String keyPassword;
+
+		public String getKeyCertChainPath() {
+			return keyCertChainPath;
+		}
+
+		public String getKeyFilePath() {
```

Comment:
IIRC this could be a file resource or something as opposed to just a string

Reply:
Yeah will figure out something to make it generic
Thanks for feedback

Reply:
Here, I've followed the same thing which is there for trusted509Certs, in similar way i've used filepath and keypath params to handle it
Need suggestion on this !!